### PR TITLE
Make external links safe by default.

### PR DESCRIPTION
**Pull request by leerob**
Closes https://github.com/shuding/nextra/issues/110.

### CODE REVIEW

**Comment by leerob** [comment]
image
Trying to get 💯 on the blog theme 😄

File: packages/nextra-theme-blog/src/mdx-theme.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ const A = ({ children, ...props }) => {
   const isExternal = props.href && props.href.startsWith('https://')
   if (isExternal) {
     return (
-      <a target="_blank" {...props}>
+      <a target="_blank" rel="noopener noreferrer" {...props}>
```
Comment (pacocoursey) [inline packages/nextra-theme-blog/src/mdx-theme.js:139]:
```suggestion
      <a target="_blank" rel="noopener" {...props}>
```

Generally just need `noopener`, as referrer can be useful for referrals!

Comment (leerob) [reply packages/nextra-theme-blog/src/mdx-theme.js]: Works for me!

**Review by shuding — APPROVED** [review]
Thanks Lee!

**Pull request merged**